### DISCUSSION TITLE
paopaodns硬件配置要求

### DISCUSSION THREAD

**Discussion by vickyleebaby** [kkkgo/PaoPaoDNS#147] [问答和帮助]
小白一名，搭建paopaodns对硬件有要求吗？就家用，预计10台设备左右使用，我搭建的是双paopaodns解析，官方文档没有标明硬件要求，特此咨询各位大神，

**Comment by kkkgo** [comment]
并没有什么特别的要求，能跑docker的设备都不会太差，容器启动会根据可用内存自动调整策略，主要看网络质量

**Reply by vickyleebaby** [reply]
感谢大神的回复，我的是一台N4100软路由搭建的IK系统，ik系统里头用虚拟机搭建的linux系统。contos7 1H3G内存，就用于docker搭建dns解析用的，就是家用解析应该都够用了吧~大神

**Reply by kkkgo** [reply]
可用内存越多调整缓存越大，足够的。
